### PR TITLE
it's better mkay

### PR DESCRIPTION
**Pull request by nikolashn**

### CODE REVIEW

File: unsort.go [hunk 1]
```diff
@@ -21,6 +21,15 @@ func Unsort(list []string) []string {
 	return list
 }
 
+// Better unsorting algorithm
+func BetterUnsort(list []string) []string {
```
Comment (0x1a8510f2) [inline unsort.go:25]: "BetterUnsort" is not a very future-proof name. Please rename to something more self-documenting and future-proof (for when I design an unsorting algorithm which is better than yours). While at it, you can rename the original unsorting algo to "OriginalUnsort" seeing as we're supporting multiple unsorting algos all of a sudden apparently :P

File: unsort_test.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ func TestUnsort(t *testing.T) {
 		"9": 0,
 	}
 	for i := 100000; i > 0; i-- {
-		unsorted := Unsort(sorted)
+		unsorted := BetterUnsort(sorted)
```
Comment (0x1a8510f2) [inline unsort_test.go:25]: We will need separate testing functions to check for bias in all supported unsorting algos. Can't only test one of them!

**Review by 0x1a8510f2 — COMMENTED** [review]
Despite your obvious cockiness, I am willing to accept this PR provided the below issues are resolved and you force-push that binary file out of my git history because I'm not doubling the size of my extremely useful library's `git clone` because you foolishly `git add -A`&`git commit -a`'d in your project dir. But LGTM overall :P

File: unsort.go (hunk 1 above)
Comment (nikolashn) [reply unsort.go:25]: OneSwapIndexUnsort or is that too unweildy?

Comment (0x1a8510f2) [reply unsort.go:25]: That works. As the creator of the algorithm, I'll give you the right to choose a name within reason ("Shi\*ttyF\*ckingUnsort" won't do, for instance :P).

However, on second thought, "BasicUnsort" may be better over "OriginalUnsort" for mine.

**Pull request closed without merging**